to-do-app/src/components/Input.js

```diff
@@ -0,0 +1,54 @@
+import React, { useReducer } from "react";
+import { useState } from "react";
+
+const Input = (props) => {
+    const [content, setContent] = useState('')
+
+    const handleChange = (event) => {
+        setContent(event.target.value)
+    }
+
+    const validateText = (text) => {
+        let isTextValid = false
+        if (text !== '') {
+            isTextValid = true
+        }
+        else {
+            isTextValid = false;
+        }
+
+        return isTextValid
+    }
+
```

Comment:
this code does the same job but more readable:
`const validateText = (text) => {
        if (!text) return false
        if (text.length<3) return false

        return true
    }`

Reply:
fixed, thanks.